.github/workflows/publish.yml

```diff
@@ -11,33 +11,22 @@ jobs:
       - name: Get sources
         uses: actions/checkout@v3
 
-      - name: Read .nvmrc for Node.js version
-        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
-        id: nvm
-
-      - name: Setup NodeJS
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          node-version: '${{ steps.nvm.outputs.NODE_VERSION }}'
+          cache: yarn
+          node-version-file: .nvmrc
           registry-url: 'https://registry.npmjs.org'
           scope: '@keeper-wallet'
-
-      - name: Install dependencies
-        run: npm ci
-
-      - name: Setup environment
-        shell: bash
-        run: |
+      - run: yarn install --frozen-lockfile
+      - run: |
           cat <<EOF > .env
-          RELAY_URL = ${{ secrets.RELAY_URL }}
-          PROJECT_ID = ${{ secrets.PROJECT_ID }}
+          VITE_RELAY_URL = ${{ secrets.RELAY_URL }}
+          VITE_PROJECT_ID = ${{ secrets.PROJECT_ID }}
```

Comment:
Why not just use `env` key for the step below? Like

```yaml
- run: yarn run build
  env:
    VITE_RELAY_URL: ${{ secrets.RELAY_URL }}
    VITE_PROJECT_ID: ${{ secrets.PROJECT_ID }}
```

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env